snuba/cli/consumer.py

```diff
@@ -135,13 +138,23 @@ def consumer(
     setup_logging(log_level)
     setup_sentry()
     logger.info("Consumer Starting")
-    storage_key = StorageKey(storage_name)
-    sentry_sdk.set_tag("storage", storage_name)
-
-    metrics_tags = {
-        "group": consumer_group,
-        "storage": storage_key.value,
+    storages = {
+        key: get_writable_storage(key)
+        for key in (StorageKey(name) for name in storage_names)
     }
+    storage_keys = [*storages.keys()]
+    # sentry_sdk.set_tag("storage", storage_name)
+
+    if len(storages) == 1:
```

Comment:
Just to confirm, it's not possible to pass 0 storages right?

Reply:
It's not possible because the CLI arg is non-optional.